src/app/messages.service.ts

```diff
@@ -35,4 +35,9 @@ export class MessagesService {
         //    like take updates while some condition is true?
     );
   }
+
+  sendMessage(newMessage:CreateMessage,conversationID:number){
+    this.httpClient.post((`http://localhost:8080/api/conversations/${conversationID}/messages`),newMessage)
```

Comment:
Please replace with a reference to environment property

Reply:
done